File: problems/0004_median-of-two-sorted-arrays/concat-sort/solution.go
func findMedianSortedArrays(nums1 []int, nums2 []int) float64 {
    nums := []int{}
    nums = append(nums1, nums2...)
    sort.Ints(nums)
    size := len(nums)
    i := size / 2
    if size % 2 == 0 {
        return float64(nums[i - 1] + nums[i]) / 2.0
    } else {
        return float64(nums[i])
    }
}

File: problems/0004_median-of-two-sorted-arrays/concat-sort/Solution.rs
impl Solution {
    pub fn find_median_sorted_arrays(nums1: Vec<i32>, nums2: Vec<i32>) -> f64 {
        let mut nums = [&nums1[..], &nums2[..]].concat();
        nums.sort();
        let size = nums.len();
        let i = size / 2;
        return if size % 2 == 0 {
            (nums[i - 1] + nums[i]) as f64 / 2.0
        } else {
            nums[i] as f64
        };
    }
}
